--- levmar.h	2011-11-30 09:07:45.000000000 +0100
+++ levmar.h	2021-07-19 09:03:59.290888943 +0200
@@ -39,7 +39,6 @@
  * Bellow, an attempt is made to issue a warning if this option is turned on and OpenMP
  * is being used (note that this will work only if omp.h is included before levmar.h)
  */
-#define LINSOLVERS_RETAIN_MEMORY
 #if (defined(_OPENMP))
 # ifdef LINSOLVERS_RETAIN_MEMORY
 #  ifdef _MSC_VER
