--- shishua/shishua-avx2.h	2021-08-25 16:43:51.051863625 +0200
+++ shishua/shishua-avx2.h	2021-08-25 16:44:23.318553205 +0200
@@ -89,7 +89,7 @@
   0x626E33B8D04B4331, 0xBBF73C790D94F79D, 0x471C4AB3ED3D82A5, 0xFEC507705E4AE6E5,
 };
 
-void prng_init(prng_state *s, uint64_t seed[4]) {
+void inline prng_init(prng_state *s, uint64_t seed[4]) {
   memset(s, 0, sizeof(prng_state));
 # define STEPS 1
 # define ROUNDS 13
--- shishua/shishua-sse2.h	2021-08-25 17:08:33.750673643 +0200
+++ shishua/shishua-sse2.h	2021-08-25 17:08:16.007309173 +0200
@@ -175,7 +175,7 @@
   0x626E33B8D04B4331, 0xBBF73C790D94F79D, 0x471C4AB3ED3D82A5, 0xFEC507705E4AE6E5,
 };
 
-void prng_init(prng_state *s, uint64_t seed[4]) {
+void inline prng_init(prng_state *s, uint64_t seed[4]) {
   // Note: output is uninitialized at first, but since we pass NULL, its value
   // is initially ignored.
   s->counter[0] = _mm_setzero_si128();
