--- shishua-avx2.h	2021-07-26 01:46:56.588238550 +0200
+++ shishua/shishua-avx2.h	2021-07-26 01:51:28.121574601 +0200
@@ -1,3 +1,4 @@
+#pragma once
 #ifndef SHISHUA_AVX2_H
 #define SHISHUA_AVX2_H
 #include <stdint.h>
@@ -88,8 +89,7 @@
   0xF06AD7AE9717877E, 0x85839D6EFFBD7DC6, 0x64D325D1C5371682, 0xCADD0CCCFDFFBBE1,
   0x626E33B8D04B4331, 0xBBF73C790D94F79D, 0x471C4AB3ED3D82A5, 0xFEC507705E4AE6E5,
 };
-
-prng_state prng_init(SEEDTYPE seed[4]) {
+prng_state inline prng_init(SEEDTYPE seed[4]) {
   prng_state s;
   memset(&s, 0, sizeof(prng_state));
 # define STEPS 1
